### PR TITLE
feat(lint): remove Spyglass warnings

### PR DESCRIPTION
**Pull request by P-Miranda**
- update some LIB modules to remove some Spyglass warnings

### CODE REVIEW

File: hardware/modules/div/iob_div_subshift/hardware/src/iob_div_subshift.v [hunk 1]
```diff
@@ -34,58 +35,72 @@ module iob_div_subshift #(
...
+       .clk_i(clk_i),
+       .cke_i(cke_i),
+       .arst_i(arst_i),
+       .data_i(pcnt_nxt),
+       .data_o(pcnt)
+   );
 
    always @* begin
-      pc_nxt      = pc + 1'b1;
+      pcnt_nxt      = pcnt + 1'b1;
```
Comment (jjts) [inline hardware/modules/div/iob_div_subshift/hardware/src/iob_div_subshift.v:76]: why change name ?

File: hardware/modules/ram/iob_ram_2p/hardware/src/iob_ram_2p.v [hunk 2]
```diff
@@ -19,20 +19,32 @@ module iob_ram_2p #(
 );
 
    //this allows ISE 14.7 to work; do not remove
-   localparam mem_init_file_int = HEXFILE;
+   localparam MEM_INIT_FILE_INT = HEXFILE;
```
Comment (jjts) [inline hardware/modules/ram/iob_ram_2p/hardware/src/iob_ram_2p.v:22]: Are you accidentally linting the memory descriptions?  Memories are not needed. But no need to undo these changes either.

File: hardware/modules/div/iob_div_subshift/hardware/src/iob_div_subshift.v (hunk 1 above)
Comment (AndreMerendeira) [reply hardware/modules/div/iob_div_subshift/hardware/src/iob_div_subshift.v:76]: to remove the length < 3 warning

File: hardware/modules/ram/iob_ram_2p/hardware/src/iob_ram_2p.v (hunk 2 above)
Comment (P-Miranda) [reply hardware/modules/ram/iob_ram_2p/hardware/src/iob_ram_2p.v:22]: yes, I was linting the memories previously
currently I use the correct wrapper hierarchy to ensure that memories are not linted
still, I think the changes to memories are ok regardless

**Review by AndreMerendeira — APPROVED** [review]
Everything seems ok
